src/frontend/efiling-frontend/src/components/input/Input.scss

```diff
@@ -0,0 +1,63 @@
+.bcgov-label {
```

Comment:
did youf steal this css from a template somewhere? Is this something we could include on the react starter repo?

Reply:
I actually did take this from the react starter repo. There were a couple of changes to the components there that I wanted to make but wasn't able to get that repo to run locally, so I copied the code here to change them in the meantime. I copied the CSS file as well just to make sure I wasn't missing anything for the Input component.

Ideally the changes to Input should be made in the react shared components repo instead.

Reply:
Lets make a reminder to do that at some point.

Reply:
shared components definitely needs some love but it would be a good chunk of time

Reply:
I've created a tech debt ticket for it, FLA-1371. I'll make a note of it in the issues of the shared components repo as well.

Reply:
I would squash that. It's technically not FLA work. Lets track the work we need to do in issues against the shared components repo.